src/hooks/useTheme.test.ts

```diff
@@ -106,38 +111,48 @@ describe("useTheme", () => {
   ])(
     "handles showSidebarLogo policy (policy: $policyValue, theme: $themeValue, expected: $expectedValue)",
     async ({ policyValue, themeValue, expectedValue }) => {
-      jest.mocked(useAsyncExternalStore).mockReturnValue({
-        data: { ...customTheme, showSidebarLogo: themeValue },
-        isLoading: false,
-      } as AsyncState);
-      jest.mocked(useManagedStorageState).mockReturnValue({
-        data: { showSidebarLogo: policyValue },
-        isLoading: false,
+      jest.mocked(themeStorage.get).mockResolvedValue({
+        ...customTheme,
+        showSidebarLogo: themeValue,
+        lastFetched: Date.now(),
```

Comment:
NIT: save the time and use it to validate against:
```suggestion
    const lastFetched = Date.now();
      jest.mocked(themeStorage.get).mockResolvedValue({
        ...customTheme,
        showSidebarLogo: themeValue,
        lastFetched,
```